corehq/ex-submodules/couchforms/getters.py

```diff
@@ -123,6 +124,13 @@ def get_date_header(request):
 
 
 def get_submit_ip(request):
+    from corehq.apps.ota.decorators import ORIGIN_TOKEN_HEADER, _test_token_valid
```

Comment:
`_test_token_valid` should be renamed to be non-private and perhaps a more explanatory name: `validate_origin_token`

Reply:
A docstring for `validate_origin_token` would also be great.